.nvmrc

```diff
@@ -1 +1 @@
-14
+16
```

Comment:
Should we use a fully qualified version, to increase determinism?

Reply:
Yes!